Übungsblatt_01/inssort.py

```diff
@@ -36,8 +36,8 @@ def inssort(inList):
             if elem < outList[j]:
                 # if yes break; j will be used to identify the index in outList
                 break
-        else:
```

Comment:
Works both. 
Here the else was meant to be only executed if the for loop runs through cleanly.
In that case, j has to be incremented. In all other cases, the for loop increments j by itself.